packages/oui-navbar/_variables.less

```diff
@@ -46,6 +46,16 @@
 @oui-navbar-menu-item-padding: rem-calc(10) rem-calc(20);
 @oui-navbar-menu-item-border-bottom: solid 1px @oui-color-silver;
 
+// Navbar Notification
+@oui-notification-info-icon-color: @oui-color-info-medium;
```

Comment:
(facultative)It's no more a requirement to create to duplicate variables in a separated file for all components